.github/workflows/ci.yaml

```
@@ -24,6 +28,74 @@ jobs:
         # See `scripts/gen-version.sh`
         fetch-depth: "0"
 
+    - name: Generate keys and allow permissions (for e2e tests)
+      working-directory: js/tests/e2e
+      run: |
+        set -x
+        openssl req -x509 -newkey rsa:4096 -keyout anbox-cloud.key -out anbox-cloud.crt -days 365 -nodes -subj "/C=GB/ST=London/L=London/O=Global Security/OU=IT Department/CN=example.com"
+        sudo chmod 0666 anbox-cloud.crt
+        sudo chmod 0666 anbox-cloud.key
+
+    - name: Install microbox (for e2e tests)
+      run: |
+        set -x
+        sudo pro attach ${{ secrets.UBUNTU_PRO_TOKEN }}
+        sudo pro enable --access-only anbox-cloud
+        sudo apt update
+        sudo apt install -y anbox-modules-dkms-121 linux-generic-hwe-22.04
```

```suggestion
        sudo apt install -y anbox-modules-dkms-122 linux-generic-hwe-22.04
```

It would be even better if we can calculate the package name based on `.base_version`